rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -134,23 +170,22 @@ type CommodityReferenceFramework: <"Specifies the type of commodity.">
 
 type DigitalAsset extends AssetBase: <"An Asset that exists only in digital form, eg Bitcoin or Ethereum, that is not backed by other Assets; excludes the digital representation of other Assets, eg coins or Tokenised assets.">
 
-type ListedDerivative extends AssetBase: <"A securitized derivative on another asset.">
-    isExchangeTraded boolean (0..1) <"Identifies whether the derivative is traded on an exchange.">
-    exchange string (0..1) <" Identifies the exchange where the asset is listed, using the scheme definition http://www.fpml.org/coding-scheme/external/exchange-id-MIC-1-0">
-        [metadata scheme]
-    exerciseTerms ExerciseTerms (1..1) <"Specifically the ExerciseDate.">
-    optionType OptionTypeEnum (0..1) <"The type of option, ie Put or Call. Left empty if it is a Future.">
-    strike OptionStrike (0..1) <"Specifies the strike of the option.">
-
-    condition ExchangeTraded: <"The Exchange must be specified for exchange-traded assets.">
-        if isExchangeTraded
-        then exchange exists
-        else exchange is absent
+type ListedDerivative extends InstrumentBase: <"A securitized derivative on another asset.">
+    expiration string (1..1) <"The Expiration Date of the contract, expressed using the same symbology as the contract identifier.">
```

Comment:
any reason why this is a string? And if it is a date should it not have "Date" in its name?

Reply:
Similar comment for Leo; an updated representation is in Phase 2